recipes/r-comics/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set version = '1.0.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-comics
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/ComICS_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/ComICS/ComICS_{{ version }}.tar.gz
+  sha256: 0af7901215876f95f309d7da6e633c38e4d7faf04112dd6fd343bc15fc593a2f
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-roxygen2
+    - r-glmnet
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-roxygen2
+    - r-glmnet
+
+test:
+  commands:
+    - $R -e "library('ComICS')"           # [not win]
+    - "\"%R%\" -e \"library('ComICS')\""  # [win]
+
+about:
+  home: http://dcq.tau.ac.il/ , http://csgi.tau.ac.il/VoCAL/
+  license: GPL-2.0-or-later
```

according to https://cran.r-project.org/web/packages/ComICS/index.html
```suggestion
  license: GPL-2.0-only
```